### ISSUE TITLE
git fsck failed on master

### ISSUE THREAD

**Issue by mikacousin** [OpenLightingProject/ola#1660]
Hello,

I want to package ola with [olc](https://github.com/mikacousin/olc) on flathub but the build failed with git fsck :

`error in tag 60784ccfa800b04ed2e754e33914a1b667e641c5: missingSpaceBeforeDate: invalid author/committer line - missing space before date`

With a git cat-file tag 60784ccfa800b04ed2e754e33914a1b667e641c5, I've got:

`object 42ce6c24c91e6773bfa564bea14f7c769e44f04`
`type commit`
`tag start`
`tagger simon <simon>`

Can someone fix this ?

**Comment by mikacousin** [comment]
Finally I package ola 0.10.7 archive in my flatpak so this issue is no longer a blocking issue for me.

**Comment by kripton** [comment]
Interesting, never heard of of `git fsck` before ...
That is basically one of the very first commits and that tag points to a commit that where the tree didn't advance further. I assume this tag + commit could be deleted without causing any harm. Mh, picture uploading seems to be broken ...

**Comment by mikacousin** [comment]
Yes, I was surprised when I discovered a `git fsck` error too ...
I just found this script : https://github.com/collectd/collectd/issues/2115#issuecomment-301118735
Seems to resolve this error (quickly tested on a local copy).

**Comment by mikacousin** [comment]
Hi,
To embed a recent version of ola with olc on flathub (problems with recent versions of python) would it be possible to delete the commit that is causing the problem?

**Comment by pdmccormick** [comment]
The repository for the [Python requests library](https://github.com/psf/requests) once had a similar issue (see [this issue](https://github.com/psf/requests/issues/2690) and also [this documentation fix](https://github.com/psf/requests/pull/5755)).

Here is a workaround: when cloning the repository, add the flag `-c fetch.fsck.missingSpaceBeforeDate=ignore`, as in:

```
git clone -c fetch.fsck.missingSpaceBeforeDate=ignore https://github.com/OpenLightingProject/ola.git

# Or:
git clone -c fetch.fsck.missingSpaceBeforeDate=ignore git@github.com:OpenLightingProject/ola.git
```

You can make this the default behaviour by setting your global config with:

```
git config --global fetch.fsck.missingSpaceBeforeDate ignore
```

**Comment by pdmccormick** [comment]
I've opened a PR to add these workaround directions to the README. This is my first time contributing to this project, so I'm not sure how exactly to go about seeking review, but hopefully someone will notice this! @kripton might that be you? Thanks!

**Comment by kripton** [comment]
Thank you @pdmccormick for your PR. While I don't oppose it, I don't think re-writing the complete history is necessary and I'd rather fix the root cause.

Since the commit in question (the one tagged with "start") doesn't have any descendants, I would simply delete the tag (or set it on the "Initial revision" commit instead) and that bad commit should then be gone from the history.

@peternewman  or @nomis52 : Any objections about me force-pushing that tag (assuming I do have the permissions)?

**Comment by kripton** [comment]
ping @peternewman

**Comment by kripton** [comment]
Polite ping @peternewman  @nomis52 : Can I please drop or force-push the tag `start` to some other commit?
